easybuild/easyblocks/c/clang.py

```diff
@@ -193,14 +193,14 @@ def prepare_step(self, *args, **kwargs):
         build_targets = self.cfg['build_targets']
         if build_targets is None:
             arch = get_cpu_architecture()
-            default_targets = DEFAULT_TARGETS_MAP.get(arch, None)
```

Comment:
How about this instead:

```python
default_targets = DEFAULT_TARGETS_MAP[arch][:]
```

the trailing `[:]` implies taking a copy of the list, so any changes to `default_targets` don't affect the values in `DEFAULT_TARGETS_MAP`